crates/pop-parachains/src/up/mod.rs

```diff
@@ -907,12 +906,12 @@ id = 1000
 chain = "asset-hub-paseo-local"
 "#
 			)?;
-			let system_parachain_version = "v1.12.0";
+			let system_parachain_version = "stable2407";
```

Comment:
Is there a reason this is `2407` instead of `2409?

Reply:
Its just testing that a different version for a system parachain is used from the relay version.